src/components/general/CommitteeForm.js

```diff
@@ -75,26 +77,18 @@ class CommitteeForm extends Component {
 
```

Comment:
In reference to line 31, can the state be split in two lines?

Reply:
In reference to line 73, add a newline character before this statement to notice the difference from ending the branching.